Automattic/jetpack
# My Jetpack: add new Blaze card and product.

## Proposed changes:

This should help surface the module and feature to new customers.

<img width="925" alt="image" src="https://github.com/Automattic/jetpack/assets/426388/76c344d6-c0f1-46e8-a813-43055c4e1121">

<img width="470" alt="image" src="https://github.com/Automattic/jetpack/assets/426388/9537629a-0d54-449d-af25-3f60a4647774">

---------

> **Warning**
> The flow doesn't work right now when the Jetpack plugin is not installed.

I'm not quite sure what is the expected flow when you use My Jetpack on a site where the Jetpack plugin is not installed (for example a site using only Jetpack Boost). The card currently displays an "Install Blaze" link (this is all handled via the existing tools in place), but that link doesn't lead anywhere. Should there be an additional interstitial offering to install the Jetpack plugin at this point, or an interstitial that offers to install *Blaze* even though that's not a plugin?

I wanted to follow the lead of the Stats card for this, but that flow is broken at the moment: https://github.com/Automattic/jetpack/issues/31343#issuecomment-1619876084

Related conversations:

- p9dueE-4g1-p2
- p9dueE-4yo-p2

### Other information:


## Jetpack product discussion

* peeHDf-1nS-p2#comment-990

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Jetpack > My Jetpack
* View the new card.
    * Ensure its status is correct (it should offer you to connect to WordPress.com if you're not connected)
    * Ensure it remains correct when toggling the module on and off in Jetpack > Settings > Traffic.
    * Check wording of the card.
    * Try clicking on the management link, see that it directs you to the right place.
* Add the following snippet to your site: `add_filter( 'jetpack_blaze_dashboard_enable', '__return_true' );`
    * Ensure that the management link keeps working.
